metadata-io/src/main/java/com/linkedin/metadata/timeseries/elastic/query/ESAggregatedStatsDAO.java

```diff
@@ -124,7 +124,7 @@ private static void rowGenHelper(final Aggregations lowestAggs, final int curLev
       }
     } else if (curLevel < lastLevel) {
       //(Recursive-case): We are still processing the nested group-by multi-bucket aggregations.
-      // For each bucket, add the key to the row and recurse-down for full row construction.
+      // For each bucket, add the key to the row and recur down for full row construction.
```

Comment:
why the rename here?

Reply:
I was taught that one should say recur instead of recurse